stdlib/Profile/src/Profile.jl

```diff
@@ -1323,7 +1327,7 @@ function _stream_heap_snapshot(prefix::AbstractString, all_one::Bool)
         end
     end
 end
-function take_heap_snapshot(all_one::Bool=false; dir::Union{Nothing,S}=nothing) where {S <: AbstractString}
+function take_heap_snapshot(all_one::Bool=false, redact_data::Bool=false; dir::Union{Nothing,S}=nothing) where {S <: AbstractString}
```

Comment:
Can we add a simple test for this? e.g. take a snapshot with/without `redact_data` and check it does/doesn't have some data we expect, something like
```julia
fname = cd(mktempdir()) do
    read(`$(Base.julia_cmd()) --startup-file=no -e "using Profile; const x = \"OH_LOOK_THIS_IS_STRING_DATA\"; print(Profile.take_heap_snapshot(redact_data=true))"`, String)
end
@test !contains(read(fname, String), "OH_LOOK_THIS_IS_STRING_DATA")
```

Reply:
Added.